src/printer.js

```diff
@@ -2158,14 +2158,65 @@ function printStatement(path, options, print) {
       ]);
     }
     case "retif": {
-      const printedExpr = concat([
+      const parts = [];
+      const parent = path.getParentNode();
+      const parentParentNode = path.getParentNode(1);
```

Comment:
If this is the parent's parent, should we call this `grandParentNode`?

Reply:
@czosel looks as good name, but in prettier and other places we use `parentParentNode`, i think better left as is